gst/tensor_decoder/tensordec.c

```diff
@@ -755,6 +761,12 @@ gst_tensordec_set_property (GObject * object, guint prop_id,
         g_assert (_tensordec_process_plugin_options (self, 0) == TRUE);
```

Comment:
change option index 1
_tensordec_process_plugin_options (self, 0)

Reply:
https://github.com/nnsuite/nnstreamer/pull/830/commits/edef2f41ea5eb5cabb5e2367c6baede1456c65cf fixes it.